larky/src/test/java/com/verygood/security/larky/modules/vgs/cerebro/text/analyzer/TextAnalyzerModuleSPITest.java

```diff
@@ -49,7 +49,7 @@ public void testNoopModule_exception() throws Exception {
     // Setup Noop Vault
     setTextPIIAnalyzerImpl("");
     System.setProperty(TextAnalyzerModule.ENABLE_INMEMORY_PROPERTY, "false");
-    textAnalyzerModule = new TextAnalyzerModule();
+    textAnalyzerModule = TextAnalyzerModule.init();
```

Comment:
you would just import INSTANCE here right? unless you want to re-instantiate it?

Reply:
Here I want to re-instantiate it, as tests need to verify how an instance behaves while initiated with different configs.

Reply:
k